src/main/java/org/jenkinsci/plugins/workflow/support/steps/input/InputStepExecution.java

```diff
@@ -236,6 +246,27 @@ public HttpResponse doAbort() {
         return HttpResponses.ok();
     }
 
+    private void notifyInput(InputExtension.InputEvent inputEvent) {
+        InputStep inputStep = this.getInput();
+        Jenkins.get().getExtensionList(InputExtension.class).forEach(input -> {
+            final String extensionName = input.getClass().getSimpleName();
+
+            User currentUser = User.current();
+            final String userID;
+            if(currentUser != null) {
+                userID = currentUser.getId();
+            } else {
+                userID = "anonymous";
```

Comment:
```suggestion
                userID = null;
```

and mark the API `@CheckForNull`